src/gui_common/CustomWindow.cs

```diff
@@ -16,6 +17,13 @@ public class CustomWindow : Control
 
     private bool hasBeenRemovedFromTree;
 
+#pragma warning disable CA2213
+    private Array<AddWindowReorderingSupportToSiblings> windowReorderingNodes = new();
+#pragma warning restore CA2213
+
+    [Signal]
+    public delegate void Dragged(CustomWindow window);
```

Comment:
This class still has functionality in it that it doesn't actually support. I think still all of the changes in your PR to this CustomWindow class should all move to CustomDialog so that half of an implementation of a feature is not in this file where it doesn't actually do anything without additional parts.